### PR TITLE
Review: button/checkbox

### PR DESCRIPTION
**Pull request by Leopoldthecoder**
Please makes sure these boxes are checked before submitting your PR, thank you!

* [ ] Make sure you follow Element's [Contributing Guide](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.md).
* [ ] Rebase before creating a PR to keep commit history clear.
* [ ] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

File: examples/docs/en-us/button.md [hunk 1]
```diff
@@ -82,22 +80,22 @@ Basic usage.
...
+The `disableds` attribute determines if the button is disabled.
 
-:::demo Use `disabled` attribute to determine whether a button is available. It accepts a `Boolean` value.
+:::demo Use `disabled` attribute to determine whether a button is disabled. It accepts a `Boolean` value.
 
 ```html
-<el-button :plain="true" :disabled="true">Primary Button</el-button>
-<el-button type="primary" :disabled="true">Primary Button</el-button>
-<el-button type="text" :disabled="true">Text Button</el-button>
+<el-button :plain="true" :disabled="true">Default Button</el-button>
```
Comment (QingWei-Li) [inline examples/docs/en-us/button.md]: 为啥这里不改成 `plan disabled`

Comment (Leopoldthecoder) [inline examples/docs/en-us/button.md]: 想告诉用户「两种写法都可以」……

Comment (QingWei-Li) [inline examples/docs/en-us/button.md]: 哦 好机智

File: examples/docs/en-us/button.md [hunk 2]
```diff
@@ -65,13 +65,11 @@
...
-Frequently-used button.
+Commonly used button.
 
-### How to use
+### Basic usage
 
-Basic usage.
-
-::: demo Button component provide 7 types of buttons, is defined by `type` attribute, and default value is `default`.
+::: demo Button provides 7 themes defined by the `type` attribute, and the default value is `default`.
```
Comment (baiyaaaaa) [inline examples/docs/en-us/button.md]: 感觉应该不加 default value is `default`，会造用户去实例里判断 vm.type === 'default' 得情况

Comment (Leopoldthecoder) [inline examples/docs/en-us/button.md]: Done.

**Pull request merged**